DataFormats/L1TrackTrigger/interface/TTBV.h

```diff
@@ -103,39 +122,49 @@ class TTBV {
     bs_ ^= bv.bs_;
     return *this;
   }
+
+  // operator: not
   TTBV operator~() const {
     TTBV bv(*this);
     return bv.flip();
   }
 
-  // operator: bit shifts
+  // operator: bit shifts right reference
   TTBV& operator>>=(int pos) {
     bs_ >>= pos;
     size_ -= pos;
     return *this;
   }
+
+  // operator: bit shifts left reference
   TTBV& operator<<=(int pos) {
     bs_ <<= S - size_ + pos;
     bs_ >>= S - size_ + pos;
```

Comment:
If shift right by "pos" after shifting left by "pos", then isn't there no net shift? 
Also, why reduce size_ by "pos" ? After left-shifting, I expect size to either increase or remain unchanged, depending on whether word length is increased to prevent bits overflowing.

Reply:
done

Reply:
Change comment to "ref. operator: remove left-hand bits"?